pkg/operator/cluster/ceph/osd/pod.go

```diff
@@ -154,9 +240,11 @@ func (c *Cluster) osdContainer(devices []rookalpha.Device, selection rookalpha.S
 		opmon.AdminSecretEnvVar(),
 		k8sutil.ConfigDirEnvVar(),
 		k8sutil.ConfigOverrideEnvVar(),
+		{Name: "PREPARE_ONLY", Value: strconv.FormatBool(prepareOnly)},
```

Comment:
can you define this as a const? Also for convention maybe you could call it `ROOK_OSD_PREPARE_ONLY`

Reply:
will fix this after rebase (since the fix will touch something outside this file)